### PR TITLE
add DeSerialize for some types in ofdb-boundary, fixup some traits an…

### PR DESCRIPTION
**Pull request by magdaddy**
…d extra-derive

### CODE REVIEW

File: ofdb-boundary/src/lib.rs [hunk 1]
```diff
@@ -714,17 +714,18 @@ impl From<e::contact::Contact> for Contact {
     }
 }
 
-#[derive(Serialize, Deserialize, Default)]
-#[cfg_attr(feature = "extra-derive", derive(Debug, PartialEq, Eq))]
+#[derive(Serialize, Deserialize)]
+#[cfg_attr(feature = "extra-derive", derive(Clone, Debug, Default, PartialEq, Eq))]
+// #[derive(Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
```
Comment (uklotzde) [inline ofdb-boundary/src/lib.rs]: Leftover comment -> delete

File: ofdb-boundary/src/lib.rs [hunk 2]
```diff
@@ -714,17 +714,18 @@ impl From<e::contact::Contact> for Contact {
...
 pub struct Links {
     #[serde(rename = "www", skip_serializing_if = "Option::is_none")]
-    pub homepage: Option<Url>,
+    pub homepage: Option<String>,
```
Comment (uklotzde) [inline ofdb-boundary/src/lib.rs]: Why do we need to reduce type-safety?

**Review by uklotzde — CHANGES_REQUESTED** [review]

**Review by uklotzde — APPROVED** [review]

File: ofdb-boundary/src/lib.rs [hunk 3]
```diff
@@ -596,24 +596,24 @@ impl From<ClearanceForPlace> for e::clearance::ClearanceForPlace {
...
+#[derive(Serialize, Deserialize)]
 #[cfg_attr(feature = "extra-derive", derive(Debug, Clone, PartialEq, Eq))]
 pub struct ResultCount {
     pub count: u64,
 }
 
 #[derive(Serialize, Deserialize)]
-#[cfg_attr(feature = "extra-derive", derive(Debug, PartialEq))]
-pub struct LatLonDegrees(f64, f64);
+#[cfg_attr(feature = "extra-derive", derive(Clone, Debug, PartialEq))]
```
Comment (flosse) [inline ofdb-boundary/src/lib.rs:606]: I think we could also derive `Copy` here

File: ofdb-boundary/src/lib.rs [hunk 4]
```diff
@@ -596,24 +596,24 @@ impl From<ClearanceForPlace> for e::clearance::ClearanceForPlace {
...
+pub struct LatLonDegrees(pub f64, pub f64);
```
Comment (flosse) [inline ofdb-boundary/src/lib.rs:607]: I think it's better to use explicit access methods.
Here is a good example:

https://github.com/kartevonmorgen/openfairdb/blob/52b939d0f6cfb37f1ea1cc7015696fecb3e2ddce/ofdb-entities/src/geo.rs#L98

**Review by flosse — CHANGES_REQUESTED** [review]

**Pull request merged**